packages/plugin-ext/src/plugin/quick-open.ts

```diff
@@ -40,6 +40,13 @@ export function isPromiseCanceledError(error: any): boolean {
     return error instanceof Error && error.name === canceledName && error.message === canceledName;
 }
 
+export class CancellationError extends Error {
```

Comment:
I just noticed that this is in quick-open.ts. However, this is a general class and should be moved to `types-impl.ts`

Reply:
Let me correct that: we're consuming cancellation-related stuff from core/common/cancellation.ts. We probably put the implementation of the class there, as well.